Projects/hCoreUI/Sources/hText/hText.swift

```diff
@@ -34,28 +34,39 @@ public enum HFontTextStyle {
     case display1
     case display2
     case display3
-    case heading3
-    case heading2
+
     case heading1
+    case heading2
+    case heading3
+
     case body1
     case body2
     case body3
+
     case label
     case finePrint
 
-    case title
-    case title1
-    case title2
-    case title3
-    case headline
-    case subheadline
-    case callout
-    case footnote
-    case standardSmall
-    case badge
+    case displayXXLShort
```

Comment:
@sladan-hedvig we could delete these and add later if necessary. What do you think?